zgw_consumers/test/factories.py

```diff
@@ -0,0 +1,26 @@
+import factory
+
+from zgw_consumers.models import Service
+
+
+class ServiceFactory(factory.django.DjangoModelFactory):
+    label = factory.Sequence(lambda n: f"API-{n}")
+    api_root = factory.Faker("url", schemes=["https"])
```

Comment:
I think the problem here is that no path is generated in the URL and there is no guarantee that the path ends with a trailing slash - this is normalized in the model at `save` time, but that obviously doesn't get called with `ServiceFactory.build()`, so the factory may generate not-representative api_root values.

I think we should bite the bullet and implement a custom faker provider that inherits from the existing one, using the `url` & generate a valid URL path with trailing slash.

Reply:
https://github.com/maykinmedia/zgw-consumers/pull/73/commits/242477896f09c435a87e905074ae92defeb94290